src/test/java/uk/gov/hmcts/darts/hearings/service/impl/HearingsServiceImplTest.java

```diff
@@ -145,6 +152,54 @@ void testGetEventsResponseCaseIsExpired() {
         assertThat(exception.getMessage()).isEqualTo("Case has expired.");
     }
 
+    @Test
+    void testRemoveMediaLinkToHearing() {
```

Comment:
Preferably go with the `methodName_expectations_conditions()` naming convention, as documented in [Best Practices](https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1694706043#BestPractices(Backend)-TestNamingShallClearlyCommunicatetheTestIntent).

Comment applies to all newly added tests in this PR.